runtime/common/src/lib.rs

```diff
@@ -66,11 +66,11 @@ parameter_types! {
     pub const TargetBlockFullness: Perquintill = Perquintill::from_percent(25);
     /// The adjustment variable of the runtime. Higher values will cause `TargetBlockFullness` to
     /// change the fees more rapidly.
-    pub AdjustmentVariable: Multiplier = Multiplier::saturating_from_rational(3, 100_000);
+    pub AdjustmentVariable: Multiplier = Multiplier::saturating_from_rational(125, 1_000_000);
```

Comment:
With this it takes 2.3844445 days to reach 1x multiplier and 3.41125 days to reach 2x
To cooldown from 1x it takes ~7 days and from 2x it takes ~10 days

1 day cost is 320k and to reach 1x it would cost ~1 million USD

Reply:
this is with the minimum multiplier set to 0.25, right?
These reaction times seem reasonable to me

I do feel the absolute number is too high tho. with KMA price rising, maybe we should target 100k / day at current price.
That would imply a lower minimum factor. The adjustment *speed* seems ok if not a bit too slow, we're talking about a fully congested chain here after all, so maybe a faster increase&cooldown would be better.
e.g. from 0.1x -> 1x in 1 day, 2x in 2 days with cooldown in 2 / 4 days respectively

Reply:
I think I have better numbers now, with some data to back it up. I'll update and organize a meeting later today or tomorrow to review.